backend/apps/candidates/models.py

```diff
@@ -0,0 +1,114 @@
+from django.db import models
+from phonenumber_field.modelfields import PhoneNumberField
+from django.utils.translation import gettext_lazy as _
+
+
+class Candidate(models.Model):
+    """
+    This class represents a Candidate model, which can be used to create
+    candidates, store their contact information and manage them. All extra
+    information is stored in the CV.
+
+    Attributes
+    ----------
+    name : str
+        a firstname of the candidate
+    surname : str
+        a surname of the candidate
+    date_of_birth : date
+        a date of birth of the candidate
+    gender : str
+        a gender of the candidate
+    phone_number : str
+        a contact number of the candidate
+    email : str
+        a contact email of the candidate
+    age : int
+        age of the candidate
+    updated_at : date
+        last time Candidate was updated
+    created_at : date
+        date of creation of the Candidate
+
+    """
+
+    class GenderChoices(models.TextChoices):
+        """This class provides enum for gender types."""
+        MALE = "M", _("Male")
+        FEMALE = "F", _("Female")
+        OTHER = "O", _("Other")
+    
+    class Meta:
+        """Standard Meta class for Candidate model."""
+        ordering = ("surname", "name")
+
+    name = models.CharField(
+        _("Firstname"),
+        max_length=50,
+        db_index=True,
+    )
+    surname = models.CharField(
+        _("Surname"),
+        max_length=50,
+        db_index=True,
+    )
+    date_of_birth = models.DateField(
+        _("Birth date"),
+        blank=True, 
+        null=True
+    )
+    gender = models.CharField(
+        verbose_name=_("Gender"),
+        max_length=1,
+        choices=GenderChoices.choices,
+        default=GenderChoices.OTHER,
+    )
+    phone_number = PhoneNumberField(
+        verbose_name=_("Phone number"),
+        blank=True,
+        db_index=True,
+    )
+    email = models.EmailField(
+        _("Email address"),
+        max_length=100,
+        blank=True,
+        db_index=True,
+    )
+    notes = models.TextField(
```

Comment:
Maybe worth delete "blank=True" in phone_number? We always need the candidate's contacts.
Email  50/50.  Its also required, but the candidate may not submit it.

Reply:
There is no ''blank=True" in phone number, it is unique and required. Do you mean to get rid of it in email?

Reply:
About phone number, I didn't see that there was a new commit that fixes "blank=True"). I think yes.